rust/private/rust.bzl

```diff
@@ -954,6 +954,12 @@ rust_shared_library = rule(
     implementation = _rust_shared_library_impl,
     attrs = dict(
         _common_attrs.items() + _experimental_use_cc_common_link_attrs.items() + {
+            "linker_script": attr.label(
+                doc = dedent("""\
+            Link script to forward into linker via rustc options.
```

Comment:
nit: could you indent this and the next line so that they both start at +4 spaces relative to where the `doc =` on the previous line starts at